script_library/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ansible==2.7.5
```

Comment:
why do we depend on a specific set of packages?

Reply:
We require a minimum version.

There is a series of tests that run to check minimal requirements in the pre-flight checks.
Should those requirements not be met, then software needs installation.

Currently the jenkins environment to deploy is non privileged, and doesn't allow software installation as root.
Additionally, the tooling is prepared in a way that's completely abstracted of the local environment (you could run this code on mac os if you want), so there is no assumption of any rpm/suse platform for the localhost.

Therefore, if the requirements are not meant, a python virtualenv is booted in user space with those requirements.

When we'll package software as rpms, our documentation will point to use said packages. (We'll probably go even further and package this code in a rpm which will have ansible as dependency).

Anyway, long story short, currently we need a requirements file to install software in a venv.

The version is bundled as static requirements, as I will introduce a bot to bump those on a regular basis.
I'd rather keep the requirements frozen, and bump when we now, instead of having a minimum requirement, as those might change and break at any time. I would rather have it updated when we want, with an auto PR (I might even setup a github bot for that).